src/universal/common/components/AppHeader/AppHeader.scss

```diff
@@ -0,0 +1,73 @@
+@import '../../../../theme/variables.scss';
+$header-height: 70px;
+
+.AppHeader {
+  height: $header-height;
+  display: flex;
+  background-color:rgba(255, 255, 255, 0.8);
+  width: 100%;
+  transition: all 0.3s ease;
+  padding: 0 8px;
+  z-index: 1;
+  box-shadow: 0 2px 2px 1px rgba(0, 0, 0, 0.1);
+
+  @media #{$screen-xs} {
+    padding: 0 36px;
+  }
+
+  * {
```

Comment:
Also the transition should be removed, there is no more animation when scrolling :).